### PR TITLE
fix-29: add draggable region and save window position

### LINKED ISSUE #29: Move app window
There seems no way to move the app window after clicking the tray icon. Causing problems with my toolbar layout with vertical toolbars on a dual monitor set up
https://gyazo.com/af148904c75839355e7b27fe5b6a0c9c

### PR DESCRIPTION
**Pull request by mikaoelitiana**
- save window position when moved
- add draggable region so window can be moved easily

fixes #29

### CODE REVIEW

**Comment by G07cha** [comment]
Whoah dude! First of all, thanks for working on this issue and second, that's a great solution because I was thinking about updating the `menubar` package but that would require more effort and time but you managed to find the better solution! 👏 

Give me some time to check it and I'll merge if everything is fine.

File: index.js [hunk 1]
```diff
@@ -49,6 +51,15 @@ mb.app.on('quit', () => {
 	mb = null;
 });
 
+mb.on('ready', () => {
+	windowState = windowStateKeeper()
+	windowState.manage(mb.window);
+})
+
+mb.on('after-show', () => {
+	mb.window.setPosition(windowState.x, windowState.y, false);
```
Comment (G07cha) [inline index.js]: We need to check here if `x` and `y` properties are actual numbers because it's undefined by default.

**Comment by G07cha** [comment]
Thanks @mikaoelitiana!

**Comment by mikaoelitiana** [comment]
You are welcome. Will you create a new release for that?

**Comment by G07cha** [comment]
I definitely will but it will take some time to build binaries for MacOS.

**Pull request merged**